submissions/Roophee/Popup_checkbox/css/main.scss

```diff
@@ -0,0 +1,422 @@
+@import "_misc/normalize.scss";
+
+$text-color: #0c4e67;
+$hover-color: #01161e;
+
+:root {
+    font-size: 16px;
+    outline: none;
+}
+
+a.active.focus,
+a.active:focus,
+a.focus,
+a:active.focus,
+a:active:focus,
+a:focus,
+button.active.focus,
+button.active:focus,
+button.focus,
+button:active.focus,
+button:active:focus,
+button:focus,
+.btn.active.focus,
+.btn.active:focus,
+.btn.focus,
+.btn:active.focus,
+.btn:active:focus,
+.btn:focus {
+    outline: 0 !important;
+    outline-color: transparent !important;
+    outline-width: 0 !important;
+    outline-style: none !important;
+    box-shadow: 0 0 0 0 rgba(0, 123, 255, 0) !important;
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+}
+
+html,
+body {
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    scroll-behavior: smooth;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: 50px 1fr 50px;
+    grid-template-areas:
+        "header"
+        "content"
+        "footer";
+}
+
+.header {
+    display: grid;
+    grid-area: header;
+    justify-items: center;
+    place-items: center;
+    background-color: #227493;
+    border-bottom: 1px solid #033142;
+    box-shadow: inset 0 0 2px #021d27;
+}
+
+.nav {
+    position: relative;
+    width: 320px;
+    height: 40px;
+    padding: 0;
+    margin: 0;
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: 100%;
+    justify-content: center;
+    align-content: center;
+}
+
+.nav-list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: grid;
+    grid-template-columns: 60px 60px 60px 60px 60px;
```

Comment:
For cases like this use the `repeat()` function, it makes your code more readable. 
[Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat()) about repeat function

Reply:
Thanks, I will. "Per aspera ad astra!"

Reply:
Thanks.  I made revisions.